index.html

```diff
@@ -361,6 +361,100 @@ <h3>We helping you find your dream location</h3>
+                                alt=""
+                                class="features__image"
+                                role="presentation"
+                                loading="lazy"
+                            />
+                        </picture>
+                    </div>
+
+                    <div class="features__content">
+                        <div class="section__heading-group features__header">
+                            <header
+                                class="section__header"
+                                aria-labelledby="features-title"
+                            >
+                                <h2 id="features-title">Key Features</h2>
+                                <h3>We offer best services</h3>
+                            </header>
+                            <p>
+                                Contrary to popular belief, Lorem Ipsum is not
+                                simply random text. It has roots in a piece of
+                                classical Latin literature from 45 BC.
+                            </p>
+                        </div>
+
+                        <div class="features__cards">
+                            <article class="features__card">
+                                <div class="features__icon-wrapper">
+                                    <i
+                                        class="icon-location features__icon"
+                                        aria-hidden="true"
+                                    ></i>
+                                </div>
+                                <div class="features__text-group">
+                                    <h4 class="features__title">
+                                        We offer best services
+                                    </h4>
+                                    <p class="features__description">
+                                        Lorem Ipsum is not simply random text
+                                    </p>
+                                </div>
+                            </article>
+
+                            <article class="features__card">
+                                <div class="features__icon-wrapper">
+                                    <i
+                                        class="icon-calendar features__icon"
+                                        aria-hidden="true"
+                                    ></i>
+                                </div>
+                                <div class="features__text-group">
+                                    <h4 class="features__title">
+                                        Schedule your trip
+                                    </h4>
+                                    <p class="features__description">
+                                        It has roots in a piece of classical
+                                    </p>
+                                </div>
+                            </article>
```

Comment:
```suggestion
                            <article class="features__card">
                                <div class="features__card-icon-wrapper">
                                    <i
                                        class="icon-calendar features__card-icon"
                                        aria-hidden="true"
                                    ></i>
                                </div>
                                <div class="features__text-group">
                                    <h4 class="features__card-title">
                                        Schedule your trip
                                    </h4>
                                    <p class="features__card-description">
                                        It has roots in a piece of classical
                                    </p>
                                </div>
                            </article>
```
we can make some BEM improvements like this